client/src/lib/dependency-injection/InjectorProvider.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import provideInjector from './provideInjector';
+
+export default provideInjector((props, context) => <div>{children}</div>);
```

Comment:
where's `children` defined? ;)

Reply:
Not sure how this got in here. Should have been deleted.